tests/lib.rs

```diff
@@ -38,7 +44,24 @@ pub fn test_markdown_html(input: &str, output: &str, smart_punct: bool, metadata
     let p = Parser::new_ext(input, opts);
     pulldown_cmark::html::push_html(&mut s, p);
 
-    assert_eq!(normalize_html(output), normalize_html(&s));
+    // normalizing the HTML using html5ever may hide actual errors
+    // assert_eq!(normalize_html(output), normalize_html(&s));
+    assert_eq!(html_standardize(output), html_standardize(&s));
+}
+
+fn html_standardize(s: &str) -> String {
+    s.trim()
+        .replace("<br>", "<br />")
+        .replace("<br/>", "<br />")
+        .replace("<hr>", "<hr />")
+        .replace("<hr/>", "<hr />")
+        .replace('\t', "")
```

Comment:
This seems to only affect `regression_test_83` where there is

```
| foo | bar  |
→|-----|------|
| baz | alef |
```
expected to render as
```
<p>| foo | bar  |
→|-----|------|
| baz | alef |</p>
```

But isn't that incorrect? The leading whitespace should be trimmed in the output.

Reply:
I think the test was incorrect, now should be right.